src/components/OrderDetailWidget/OrderDetailWidget.tsx

```diff
@@ -52,10 +59,22 @@ const OrderDetailWidget: FC = () => {
         onChangeTokenClick={() => {}}
         onMaxButtonClick={() => {}}
       />
+      <StyledInfoButtons
+        ownerIsCurrentUser
+        onFeeButtonClick={toggleShowFeeInfo}
+      />
       <ActionButtons
         onBackButtonClick={handleBackButtonClick}
         onSignButtonClick={() => {}}
       />
+
+      <Overlay
+        title={t("common.fee")}
+        onCloseButtonClick={() => toggleShowFeeInfo()}
+        isHidden={!showFeeInfo}
+      >
+        <FeeModal />
```

Comment:
FeeModal has no Back button while OrderTypesModal does. Could add one for consistency.